node_modules/oae-admin/tenants/tenants.html

```diff
@@ -19,6 +19,7 @@
                 {else}
                     <th>__MSG__HOST__</th>
                 {/if}
+                <th>__MSG__EMAIL_DOMAIN__</th>
```

Comment:
I'd prefer not to add the email domain to the main tenants list, as the tenants list is already short on vertical space. Wouldn't it be easier to just make the email domain configuration a standard `globalAdminOnly` configurable field?